integration-tests-ex/cases/src/test/java/org/apache/druid/testsEx/auth/ITSecurityBasicQuery.java

```diff
@@ -154,6 +161,9 @@ public void testIngestionWithPermissions() throws Exception
     );
     securityClient.setPermissionsToRole(ROLE_1, permissions);
 
+    // Wait for a second so that the auth is synced, to avoid flakiness
+    Thread.sleep(1000);
```

Comment:
I am not sure if there is a better way to check this. 
We can do a follow up if we find this breaking in future releases.